src/common/libkvs/kvs_checkpoint_private.h

```diff
@@ -0,0 +1,33 @@
+/************************************************************\
```

Comment:
Since there isn't a conflicting public version of this header like say `message.h` in libflux, maybe this can just be called `kvs_checkpoint.h`?